app/default/application_routes.py

```diff
@@ -179,6 +180,16 @@ def tasklist(application_id):
             )
         ),
     }
+    app_guidance = ""
```

Comment:
Ppicky but should this be "app_guidance = None", using None to represent absence or missing value.

Reply:
Yep, good shout 👍